.github/workflows/bbw_build_container_rhel.yml

```diff
@@ -73,6 +73,9 @@ jobs:
         run: |
           cd ${{ env.WORKDIR }}
           cat ${{ matrix.dockerfile }} qpress.Dockerfile buildbot-worker.Dockerfile >$GITHUB_WORKSPACE/Dockerfile
+          if [ "${{ matrix.nogalera }}" == true ]; then
```

Comment:
For RHEL this is will always evaluate to false. This is what you intended, right?